cli/githubutil.py

```diff
@@ -186,6 +205,36 @@ def release_and_prepare_next_dev_cycle(
         commit_message="Prepare next dev cycle " + next_version_dev
     )
 
+def post_to_slack(
+    slack_cfg_name: str,
+    channel: str,
+    title: str,
+    message: str,
+):
+    api_token = ctx().cfg_factory().slack(slack_cfg_name).api_token()
```

Comment:
please stick to the pattern as used for other "Helpers"; create a class, which in turn is initialised with passed "configuration" object (SlackConfiguration in your case).